### ISSUE TITLE
negative y values for maxheight ignored

### ISSUE THREAD

**Issue by DefectiveNu** [webbukkit/dynmap#3569]
**Issue Description:** creating a custom perspective with maximum height in the negatives causes it to render as if its not set

* **Dynmap Version:** v3.3-beta-2
* **Server Version:** paper 1.18.1 build 77
* **Pastebin of Configuration.txt:** https://pastebin.com/700iVGMq
* **Server Host (if applicable):** self-hosted
* **Other Relevant Data/Screenshots:** https://github.com/webbukkit/dynmap/blob/v3.0/DynmapCore/src/main/java/org/dynmap/hdmap/IsoHDPerspective.java#L1237
* **Steps to Replicate:** create and use a custom perspective with `maximumheight: -10` or any negative 

 [x] *I have looked at all other issues and this is not a duplicate*  
 [x] *I have been able to replicate this*

**Comment by lazaroblanc** [comment]
Hi, I have the same issue

I have set-up two custom perspectives for diamond and copper mines

```yaml
  - class: org.dynmap.hdmap.IsoHDPerspective
    name: iso_S_90_lowres_diamondmines
    azimuth: 180
    inclination: 90
    scale: 4
    minimumheight: -59
    maximumheight: -56
  - class: org.dynmap.hdmap.IsoHDPerspective
    name: iso_S_90_lowres_coppermines
    azimuth: 180
    inclination: 90
    scale: 4
    minimumheight: 46
    maximumheight: 49
```

The copper mines render just fine, the diamond mines render everything. The values for min/maxheight are ignored.

Copper mines:

![image](https://user-images.githubusercontent.com/40198445/146644661-40978be1-bbec-4179-9ed1-fd492dff88f1.png)

Diamond mines:

![image](https://user-images.githubusercontent.com/40198445/146644686-867f061e-101b-4c1a-bf6e-db69e8895eb9.png)

**Comment by mikeprimm-triva** [comment]
Should be fixed in 3.3-beta-3
